Package@swift-5.4.swift

```diff
@@ -0,0 +1,55 @@
+// swift-tools-version:5.4
```

Comment:
And we are effectively dropping support for Swift < 5.4?

Reply:
yea that is the intention